doc/vector/riscv-crypto-vector-element-groups.adoc

```diff
@@ -1,7 +1,7 @@
 [[crypto-vector-element-groups]]
 === Crypto Vector Element Groups
 
-Many vector crypto instructions operate on larger operands than instructions in the base vector extension. Typically, these operands are 128- and 256-bits wide. In many cases, these operands are comprised of smaller elements that are combined. However, in other cases these operands are a single value. For example, AES operates on 128-bit blocks comprised of four 32-bit words using 128-bit round keys that are generated from 128-, 192-, and 256-bit cipher keys.
```

Comment:
"bits" is correct in this case

Reply:
@kdockser I was not sure of this correction, but since we use `with the 32- and 64-bit element widths` later in this section I figure the `s` was superfluous. I will revert in a future PR.
Are both syntax correct or does it need to always be <number>-, ..., <number'>-<plural> ?